static/app/views/replays/detail/breadcrumbs/index.tsx

```diff
@@ -126,6 +134,19 @@ function Breadcrumbs() {
       <FilterLoadingIndicator isLoading={isFetchingExtractions || isFetchingTraces}>
         <BreadcrumbFilters frames={frames} {...filterProps} />
       </FilterLoadingIndicator>
+      {isDismissed ? null : (
+        <Alert
+          type="info"
+          showIcon
+          trailingItems={<StyledCloseButton onClick={dismiss} />}
```

Comment:
This is not a11y friendly. 

I think `<Button icon={<IconClose />} onClick={dismiss} />` is the right combo.

Reply:
with whatever the correct `size=` if needed